WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/EditAttributesViewController.swift

```diff
@@ -74,19 +102,13 @@ extension EditAttributesViewController {
 extension EditAttributesViewController: UITableViewDataSource, UITableViewDelegate {
 
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return 6 // Temporary
+        return viewModel.attributes.count
```

Comment:
Just a thought. Would it make sense to expose the number of attributes as a property of the view model? Maybe even provide a getter, some short of attributeAt(indexPath), so that the view controller does not need to deal with the fact that attributes are packed in an array?

Reply:
I didn't do that because the data structure is very simple and doesn't seem prone to change, so I didn't see much benefit. I could add them if you feel strongly about it tho! :-)